site/_includes/docs/v1/tech/tutorials/_integrate-intro.liquid

```diff
@@ -0,0 +1,9 @@
+In this tutorial, you are going to learn how to integrate a {{ include.technology }} application with FusionAuth.
```

Comment:
What a same, because I've stopped using this file in favor of the existing `.md` that we used in the React file. Maybe we could change it later to allow a more specific customization?